packs/pathfinder-monster-core/aeolaeka.json

```diff
@@ -1053,7 +1053,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The aeolaeka conjures churning stones, creating a @Template[type:line|distance:60] of rolling boulders. Creatures in the line take @Damage[10d6[bludgeoning]] damage with a @Check[type:reflex|dc:35] save. The area is difficult terrain for 24 hours before the leftover stone crumbles to dust. The aeolaeka can't Liberate the Earth for [[/br 1d4 #Recharge Liberate the Earth]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature takes no damage.</p>\n<p><strong>Success</strong> The creature takes half damage.</p>\n<p><strong>Failure</strong> The creature takes full damage and is knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone].</p>\n<p><strong>Critical Failure</strong> The creature takes double damage, is knocked prone, and is @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] by the rubble (@UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC 32).</p>"
+                    "value": "<p>The aeolaeka conjures churning stones, creating a @Template[type:line|distance:60] of rolling boulders. Creatures in the line take @Damage[10d6[bludgeoning]] damage with a @Check[type:reflex|dc:35] save. The area is difficult terrain for 24 hours before the leftover stone crumbles to dust. The aeolaeka can't Liberate the Earth for [[/br 1d4 #Recharge Liberate the Earth]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature takes no damage.</p>\n<p><strong>Success</strong> The creature takes half damage.</p>\n<p><strong>Failure</strong> The creature takes full damage and is knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone].</p>\n<p><strong>Critical Failure</strong> The creature takes double damage, is knocked prone, and is @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] by the rubble ([[/act escape dc=32]]{Escape (DC 32)}).</p>"
```

Comment:
The label inside the curly bracers should be unnecessary in this case, as it matches the default label.